src/components/Callout/index.js

```diff
@@ -1,23 +1,17 @@
 import * as React from "react";
-import {
-  H3,
-  H2,
-  H1,
-  PrimaryButton,
-  ButtonText,
-  GREEN,
-} from "../../styles/styles";
-import { Container } from "./styled";
+import { H2, GREEN } from "../../styles/styles";
+import { Container, LargeButton } from "./styled";
+import Button from "../Button";
 
 const CallOut = ({ label, header }) => (
   <>
     <Container>
       <H2>{header}</H2>
-      <PrimaryButton color={GREEN}>
-        <ButtonText>
-          <H1>{label}</H1>
-        </ButtonText>
-      </PrimaryButton>
+      <LargeButton>
+        <Button to={`/submit`} color={GREEN} jumbo="true">
```

Comment:
this can be simplified just to `jumbo`, you don't need the true string/to set it equal to anything bc the logic within the button is just checking for the existence of that variable!